ts/third_party/sourcemapper/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Given that this file is relatively small, can we just copy the content that we need as opposed to bringing via third_party? This is Google's code so it's not third party to us.

Reply:
SGTM.

I have a comment indicating where it was copied from, which is probably sufficient.